### ISSUE TITLE
ngx-timepicker-field, uppendToInput and enableKeyboardInput

### ISSUE THREAD

**Issue by krVoid** [Agranom/ngx-material-timepicker#384]
Hi,

In the ngx-material-timepicker is option to change the time using the timepicker (with the keyboard), as well as an option to append a timepicker to the input (enableKeyboardInput and appendToInput).
![image](https://user-images.githubusercontent.com/38069723/113899308-f5545780-97cc-11eb-9ae1-f28ebb2fa078.png)

 Unfortunately there is no way to set these options in ngx-timepicker-field.

Is there any other option to pin a timepicker to the input using ngx-timepicker-field?

Thanks for your component.

Best regards,
Kinga

**Comment by PiotrWegrzyn** [comment]
Yes, attaching the timepicker to the input would be so useful, I was surprised to find that there is no option to do so. Pls add this
